File: Epam.HostFiles.Web/Scripts/models/userInfo.js
$(function () {
    appHostFiles.UserInfo = Backbone.Model.extend({
        defaults: {
            Name: '',
            Login: '',
            Password: '',
            UserRole: ''
        }
    });

    appHostFiles.UserInfoCollection = Backbone.Collection.extend({
        model: window.appHostFiles.UserInfo,
        url: 'api/users'
    });
})

File: Epam.HostFiles.Model/Models/UserInfo.cs
namespace Epam.HostFiles.Model.Models
{
    public class UserInfo
    {
        public int UserInfoId { get; set; }
        public string Name { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        
        public int UserRoleId { get; set; }
        public virtual UserRole UserRole { get; set; }
    }
}
